server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/BallotManifestUpload.java

```diff
@@ -198,8 +200,10 @@ private void parseAndPersistFile(final UploadInformation the_info) {
         }
         Main.LOGGER.info(parser.parsedIDs().size() + 
                          " ballot manifest records parsed from upload file");
-        Main.LOGGER.info(BallotManifestInfo.getMatching(null).size() + 
-                         " uploaded ballot manifest records in storage");
+        final long count = count();
+        if (count >= 0) {
```

Comment:
Yeah, I mean, that's fair, because if we were on 7 it wouldn't even be a choice. Might be nice for us, though.